DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/GrapesJsManagers/editor.js

```diff
@@ -256,6 +256,19 @@ $(document).ready(function () {
         global.vjEditorSettings = data;
 
         if (isEditPage()) {
+
+            if (!data.EditPage) {
+
+                $(document).on("click", function () {
+                    VjEditor.select();
+                    ShowBlockUI();
+                });
+
+                $('#vjEditor').scroll(function () {
```

Comment:
Shouldn't there be a debounce here? Otherwise, this will make the UI extremely slow.

Reply:
I have added debounce. Please review it.